bootstrap/config/kfctl_gcp_iap_master.yaml

```diff
@@ -0,0 +1,205 @@
+# Please set project and email!
```

Comment:
How is this different from?
https://github.com/kubeflow/kubeflow/blob/master/bootstrap/config/kfctl_gcp_iap.0.6.yaml

Should we organize these in a GCP directory?

Per comment above can we use KfDef.Spec.Applications and not KfDef.Spec.componentParams/components/packages

Reply:
I thought we want one for 0.6?
And future change will be made to the master one.

Reply:
The config for 0.6 should now be on the v0.6-branch.

The reason I named it kfctl_gcp_iap.0.6.yaml was to distinguish it from some of the other configs in this directory which are for 0.5 and click to deploy.

I think on master we only want one KfDef YAML for gcp + IAP. So I think we should pick kfctl_gcp_iap_master.yaml or kfctl_gcp_iap.0.6.yaml and delete the other one.

Probably keep kfctl_gcp_iap_master.yaml since its been updated to use KfDef.Spec.Applications.

We could probably change the name and remove "master". The version can be conveyed by the branch its on; we don't need it in the name.